src/plugins/github/parser.js

```diff
@@ -338,8 +348,12 @@ class GithubParser {
       payload: repositoryPayload,
     };
     this.graph.addNode(repositoryNode);
-    repositoryJSON.issues.nodes.forEach((i) => this.addIssue(i));
-    repositoryJSON.pullRequests.nodes.forEach((pr) => this.addPullRequest(pr));
+    repositoryJSON.issues.nodes.forEach((i) =>
```

Comment:
nit: can we call the loop variable `issue`? At a glance, this confused me because I read `i` as `index`.